setup.py

```diff
@@ -2,6 +2,10 @@
 
 from setuptools import find_packages, setup
 
+AVRO_REQUIREMENTS = ["avro>=1.11.1"]
+JSON_REQUIREMENTS = ["python-rapidjson>=1.8", "fastjsonschema>=2.16.2"]
```

Comment:
I would put these inline -- I don't think the constants give any additional benefits here